### PR TITLE
Add target platform to configuration profile

### PR DESCRIPTION
**Pull request by epostea**
Add the template cloud provider in the configuration profile as `target_platform`.

### CODE REVIEW

File: app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb [hunk 1]
```diff
@@ -6,10 +6,12 @@ def parse
 
   def configuration_profiles
     collector.templates.each do |template|
+      target_platform = template["manifest"]["template_provider"] if defined? template["manifest"]
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb:9]: You can do `template.dig("manifest", "template_provider")` if you're worried about `template["manifest"]` being nil

**Review by agrare — APPROVED** [review]

File: app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb (hunk 1 above)
Comment (epostea) [reply app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb]: Nice, thanks for the tip

**Comment by agrare** [comment]
Awesome @epostea, once we start displaying this on the UI can you come back and post a screenshot on this PR?

**Comment by k-chu** [comment]
ManageIQ-Add-Description-Target-Platform-Configuration-Profile-2

**Pull request merged**